src/2_RTB/VerbaleInterno_071123/content.typ

```diff
@@ -20,39 +20,39 @@ Su proposta dei componenti è stata assegnata questa prima distribuzione dei ruo
+Riccardo Toniolo, nel ruolo di responsabile, si occupa della redazione e della presentazione del prossimo #glossary("DDB") in data 13/11/2023.
 
 == Corrispondenza con Sync Lab
-A seguito dell'assegnazione del capitolato, si è deciso di contattare l'azienda Sync Lab per chiedere un incontro organizzativo e di chiarimento. L'azienda ha risposto positivamente alla richiesta e ha proposto un incontro in data 10/11/2023 alle ore 10:00. L'incontro si terrà in modalità remota tramite la piattaforma _Google Meet_.
+A seguito dell'assegnazione del capitolato, si è deciso di contattare l'azienda Sync Lab per chiedere un incontro organizzativo e di chiarimento. L'azienda ha risposto positivamente alla richiesta e ha proposto un incontro in data 10/11/2023 alle ore 10:30. L'incontro si terrà in modalità remota tramite la piattaforma _Google Meet_.
 
 == Modifica del versionamento
 Si è deciso di modificare il versionamento del progetto, in risposta ai suggerimenti del committente. Il nuovo versionamento avrà la forma X.Y, dove:
 - X viene cambiato una volta che il documento è pronto per la revisione di avanzamento;
 - Y viene cambiato ad ogni modifica successiva apportata dai redattori.
-La responsabilità della modifica delle sezione Versionamento nelle #glossary("NdP") è stata assegnata a Matteo Rango.
+La responsabilità della modifica della sezione Versionamento nelle #glossary("NdP") è stata assegnata a Matteo Rango.
 
 == Modifica dei nomi dei documenti
-In seguito ai suggerimenti del committente, si è deciso di modificare i nomi dei documenti, in modo che contengano la versione attuale. Per coerenza, verranno rinominati i documenti pertinenti alla candidatura. Inoltre, sempre sotto suggerimento del committente, verranno rimosse, dalle tabelle change log, le entry relative alle revisioni. La modifica dei nomi, oltre che il riadattamento dei documenti al nuovo sistema di versionamento e la modifica dei change log, saranno attuati da Nancy Kalaj.
+In seguito ai suggerimenti del committente, si è deciso di modificare i nomi dei documenti, in modo che contengano la versione attuale. Inoltre, sempre sotto suggerimento del committente, verranno rimosse, dalle tabelle change log, le entry relative alle revisioni.
 #pagebreak()
 
 == Sostituzione note taking system
-Attualmente il Notion condiviso viene utilizzato per prendere note relative ai meeting, all'organizzazione dell'_ordine del giorno_ per questi ultimi e per l'allocazione delle risorse temporali dei componenti. Tuttavia, la sua limitatezza a 1000 _blocchi_, con il piano gratuito attualmente utilizzato, rischia in un futuro di non essere più sufficiente. Per questo motivo è stata aperta una issue sul nostro ITS, che ha lo scopo di trovare un sistema valido che possa sostituirlo. La responsabilità di trovare un sistema alternativo è stata assegnata a Giacomo D'Ovidio e Riccardo Toniolo.
```

Comment:
toglierei l'articolo determinativo davanti a Notion: lascerei solo attualmente notion viene utilizzato per .... e in generale come strumento di condivisione di idee e informazione.